<ui version="4.0" >
 <class>Sepa20022Base</class>
 <widget class="QWidget" name="Sepa20022Base" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Generacion SEPA</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>5</number>
   </property>
   <property name="margin" >
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>5</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="minimumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../../bulmages.qrc" >:/Images/bank-list.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame" >
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>5</number>
        </property>
        <property name="spacing" >
         <number>5</number>
        </property>
        <item row="0" column="3" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>291</width>
            <height>31</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Banco</string>
       </property>
       <property name="buddy" >
        <cstring>mui_idbanco</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BfBuscarBanco" name="mui_idbanco" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="title" >
      <string>Fecha de Cargo</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QDateEdit" name="mui_fecha" >
        <property name="displayFormat" >
         <string>dd/MM/yyyy</string>
        </property>
        <property name="calendarPopup" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mui_fechafac" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Fecha de factura</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="mui_dias" />
      </item>
      <item>
       <widget class="QComboBox" name="mui_unidad" >
        <property name="insertPolicy" >
         <enum>QComboBox::NoInsert</enum>
        </property>
        <item>
         <property name="text" >
          <string>Dias</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Semanas</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Meses</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Anos</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Concepto</string>
       </property>
       <property name="buddy" >
        <cstring>mui_concepto</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mui_concepto" >
       <property name="toolTip" >
        <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Patron del texto que recibiran los clientes de su banco como concepto. El concepto sera el texto introducido aqui sustituyendo los siguientes parametros por su valor para cada factura&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">%serie&lt;/span> - Serie de la factura&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">%num&lt;/span> - Numero de facturra&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">%id&lt;/span> - identificador interno de la factura&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">%fechafac[&lt;/span>&lt;span style=" font-weight:600; font-style:italic;">dd.MM.yyyy&lt;/span>&lt;span style=" font-weight:600;">]&lt;/span> - fecha de factura en algun formato aceptado por postgresql&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">%fecharec[&lt;/span>&lt;span style=" font-weight:600; font-style:italic;">dd.MM.yyyy&lt;/span>&lt;span style=" font-weight:600;">]&lt;/span> - fecha de cargo del recibo en algun formato aceptado por postgresql (puede coincidir o no con la fecha de la factura, segun lo que indique en Fecha de Cargo)&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">%desc&lt;/span> - Texto de descripcion de la factura&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">%ref&lt;/span> - referencia de la factura.&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">%base&lt;/span> - base imponible&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">%imp&lt;/span> - impuestos&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">%total&lt;/span> - total con impuestos incluidos&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">%coment&lt;/span> - comentario de la factura&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">%contact&lt;/span> - persona de contacto&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;/body>&lt;/html></string>
       </property>
       <property name="text" >
        <string>FRA %serie%num %desc </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Archivo</string>
       </property>
       <property name="buddy" >
        <cstring>mui_archivo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mui_archivo" />
     </item>
     <item>
      <widget class="QToolButton" name="mui_buscararchivo" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mui_aceptar" >
       <property name="text" >
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mui_cancelar" >
       <property name="text" >
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BfBuscarBanco</class>
   <extends>QComboBox</extends>
   <header>bfbuscarbanco.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mui_idbanco</tabstop>
  <tabstop>mui_fecha</tabstop>
  <tabstop>mui_fechafac</tabstop>
  <tabstop>mui_dias</tabstop>
  <tabstop>mui_unidad</tabstop>
  <tabstop>mui_concepto</tabstop>
  <tabstop>mui_archivo</tabstop>
  <tabstop>mui_buscararchivo</tabstop>
  <tabstop>mui_aceptar</tabstop>
  <tabstop>mui_cancelar</tabstop>
 </tabstops>
 <resources>
  <include location="../../../bulmages.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>mui_fechafac</sender>
   <signal>toggled(bool)</signal>
   <receiver>mui_fecha</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>79</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel" >
     <x>190</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
